Makefile.include

```diff
@@ -3,6 +3,10 @@
 
 all:
 
+all-debug: export CFLAGS += -g -O0
```

Comment:
I think adding `-gdwarf-3` (4 and 5 are not supported by all distributions yet) would be nice to resolve macros to values :-)